pkgs/servers/fishnet/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, stdenv
+, rustPlatform
+, callPackage
+, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "fishnet";
+  version = "2.2.3";
+
+  src = fetchFromGitHub {
+    owner = "niklasf";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "159fwjy70n6lvnhdwv65azgi03r5qcc2m2zpzgz0k3r6cy06faxj";
+  };
+
+  assets = callPackage ./assets.nix { };
+
+  cargoSha256 = "1bfs8dy08799r6d63sb33zwcxas3gzp7jvcxv3w8n64gffan8f2n";
+
+  preBuild = ''
+    rmdir ./assets
+    ln -snf ${assets}/${assets.relAssetsPath} ./assets
+  '';
+
+  meta = with lib; {
+    description = "Distributed Stockfish analysis for lichess.org";
+    homepage = "https://github.com/niklasf/fishnet";
+    license = licenses.gpl3;
```

Comment:
Please add at least maintainers.

Reply:
Will do. It is a draft after all.

Reply:
Also gpl3Only or gpl3Plus.

I'll add it anyway because so neither me or you will forget it.

Reply:
Assets are stockfish which is licensed gpl3Only. Fishnet itself is gpl3Plus.